community/roadmap.qmd

```diff
@@ -4,22 +4,73 @@ title: Roadmap
-```
-
-Acronyms used in the Gantt chart are: API for Application Programming
-Interface; CRUD for Create, Read, Update, Delete; UI for User Interface;
-and, MVP for Minimum Viable Product. Milestones are shown as diamonds.
+While the core deliverable of Seedcase is the Product described in the [design](/design/index.md) documentation, there are several deliverables that form this core Product.
 
+Because of the evolving nature of software, they never are "completed" as they continue to be improved on and updated. So here we define deliverable for software as the first Minimally Viable Product (or prototype). After the first release of a Minimally Viable Product, the Product continues to be tested and refined based on user feedback, identifying bugs, and adding features.
+
+Likewise, our non-software projects will also continue to be refined and updated. So we define deliverable as the first time they are uploaded to an archive (like a software release) to create a DOI (digital object identifier).
+
+## Primary deliverables
+
+<!-- TODO: Convert this into a table? -->
+
+- Software
+    - Authenticating users and managing permissions
+    - Uploading and managing research data
+        - Plugin/extension system for processing or enriching data
+    - Cataloging and browsing metadata on data in a Data Resource
+    - Managing and displaying Data Projects (`seedcase-registry`)
+    - Submitting request for accessing specific data
+    - Web portal for displaying basic details on a Data Resource
+    - Plugin/extension system for theming of User Interface elements
+
+- Documents
+    - Contributing to and maintaining Seedcase software
+    - Using and interacting with Seedcase software through their API and the command line
+    - Using and interacting with Seedcase software through the User Interface
+    - Software and data design and architecture for each Seedcase software
+    - Installing and deploying Seedcase software
+    - Usin customizing Seedcase software
+
+- Workshops (in-person and maybe virtual as videos)
+    - Installing and setting up Seedcase on a server so Data Resources can be created
+    - Using Seedcase to create and manage a Data Resource
+    - Requesting to start a data project for a Data Resource
+
+- Sub-projects
+    - Workflows and digital infrastructure for building software in a team-based research environment
+        - Report
+        - Documentation
+        - Templates
+
+## Secondary deliverables
+
+- General sharing
+    - Knowledge and skills learned from attending conferences and workshops in research software and data engineering or operational management
+
+- Awareness/education
+    - "What is research data engineering?"
+
+- Training and learning
+    - Workshop: "Data engineering in a research environment
+
+## Major milestones
+
+- First upload of documents to archive (for DOI)
+    - Design and architecture
+    - Team workflow and infrastructure
+
```

Comment:
Major milestones could include each module in the Seedcase software (when they have been mapped out)?